cloudflared/config.yaml

```diff
@@ -36,3 +36,7 @@ schema:
   reset_cloudflared_files: bool?
   data_folder: list(config|share|ssl)?
   custom_config: bool?
+  warp_enable: bool?
+  warp_routes:
```

Comment:
I cannot test right now, but from my understanding, once you add an option with an array like this, the array always needs to be present in the config then, as an empty array. That's why we have to add "additional_hosts: []" always, even if we don't need it.
Did you check, if the add-on continues to work without that option? If not as I expect, I think we should add the empty array above in the options part to not break the standard flow.

Reply:
I have tested this and it works, I would guess because it is just a string list and not a list nesting objects. 

See schema from [Dev Docs](https://developers.home-assistant.io/docs/add-ons/configuration)

```
message: str
logins:
  - username: str
    password: str
random:
  - "match(^\\w*$)"      <--- 
link: url
size: "int(5, 20)"
count: float
not_need: "str?"
```

Reply:
ok, perfect 